packages/app-page-builder-elements/src/renderers/form/types.ts

```diff
@@ -87,6 +105,8 @@ export type FormLayoutComponentProps<T = any> = {
     getDefaultValues: () => { [key: string]: any };
     goToNextStep: () => void;
     goToPreviousStep: () => void;
+    validateStepConditions: (formData: Record<string, any>, stepIndex: number) => void;
```

Comment:
We get `formData` from `Form` component and by default the type of that data is `any`.
But the `formData` contains form fields and their values, so I will investigate whether we can use `keyof typeof` in order to create a proper type.

Reply:
After investigation I can say that this should probably stay as `Record<string, any>` because pararm `data` of property `onChange` of `Form` component has type `Record<string, any>`. So there not much we can do to improve type

Reply:
OK!